pyriemann/utils/covariance.py

```diff
@@ -11,41 +11,69 @@
 
 def _lwf(X, **kwds):
     """Wrapper for sklearn ledoit wolf covariance estimator"""
-    if not is_real_type(X):
-        raise ValueError("Input must be real-valued.")
+    iscomplex = np.iscomplexobj(X)
+    if iscomplex:
+        X = np.concatenate((X.real, X.imag), axis=0)
     C, _ = ledoit_wolf(X.T, **kwds)
+    if iscomplex:
+        C = _make_complex_covariance(C)
```

Comment:
you can do this with a tiny decorator to avoid the copy / paste below in all functions

Reply:
ah sorry, I always forget about python decorators. will look into it.

Reply:
Not sure if decorators need documentations, but I added a few lines.